### DISCUSSION TITLE
Artefacted fonts

### DISCUSSION THREAD

**Discussion by fruktiz32** [Bodmer/TFT_eSPI#1130] [Q&A - Fonts]
Hello, I just connected my 3.5" display to ESP32 and get artefacted font. What I'm doing wrong? I chose ILI9486 driver, but I'm not sure if it is correct one. 
https://ibb.co/wM4kj7L

**Comment by Bodmer** [comment]
I am guessing a SPI interface. ILI9486 displays cannot be over clocked much, try 12 MHz,

**Reply by fruktiz32** [reply]
I just figured out it is broken hardware, because if I bent TFT display everything works fine, but in default it shows artefacts sometimes. You can close discussion. Thanks!

**Reply by Bodmer** [reply]
Great.
